app/src/main/java/com/github/se/studybuddies/MainActivity.kt

```diff
@@ -149,9 +150,9 @@ class MainActivity : ComponentActivity() {
                 arguments = listOf(navArgument("groupUID") { type = NavType.StringType })) {
                     backStackEntry ->
                   val groupUID = backStackEntry.arguments?.getString("groupUID")
-                  ifNotNull(groupUID) { groupUID ->
+                  ifNotNull(groupUID) { groupUid ->
                     val groupViewModel = remember { GroupViewModel(groupUID, db) }
-                    GroupScreen(groupUID, groupViewModel, chatViewModel, navigationActions, db)
+                    GroupScreen(groupUid, groupViewModel, chatViewModel, navigationActions, db)
```

Comment:
Thanks for the change, next time you could use another more explicit name or maybe do it in one line :
`ifNotNull(backStackEntry.arguments?.getString("groupUID")) { groupUID ->}`